protocol/contracts/zaps/TempleZaps.sol

```diff
@@ -232,7 +232,8 @@ contract TempleZaps is ZapBase {
     address token1 = IUniswapV2Pair(pair).token1();
 
     if (_fromAddress != token0 && _fromAddress != token1) {
-
+      require(approvedTargets[_fromAddress][_swapTarget], "TempleZaps: Unsupported token/target");
```

Comment:
@princetonbishop It's a little odd that the `approvedTargets` are on this TempleZaps (via the ZapBase inheritance) too, but currently unused.

What were you thinking here - that TempleZaps has it's own allow list, or that it shares the same list as GenericZap (in which case we'd change this line to use `zaps.approvedTargets()`?

Reply:
Also, I can't see any tests which check that the unsupported token/target fails...might be worth setting up solidity coverage in temple core?

Reply:
for consistency across zaps, making `templeZaps` use zaps approved targets is more ideal.

Reply:
Works for me -- should probably remove from ZapBase and put directly into GenericZaps then I think